platform/security/certificate/security-certificate-generator/src/main/java/org/codice/ddf/security/certificate/generator/CertificateCommand.java

```diff
@@ -26,68 +28,105 @@
 
 public class CertificateCommand {
 
+    private static final String SAN_EXAMPLE = "-san DNS:localhost,IP:127.0.0.1";
+
     /**
      * Pass in a string to use as the common name of the certificate to be generated.
-     * Exception thrown if 0 arguments or more than 1 argument.
+     * Exception thrown if 0 arguments or more than 2 argument (or more than 4 when -san is used).
      *
      * @param args
      */
     public static void main(String args[]) {
-        if (args.length != 2) {
+        // try to extract -san if provided
+        int expected = 2;
+        String[] sans = null;
+        int mainIndex = 0;
+
+        if (args.length == 4) {
+            if (args[0].trim()
+                    .equalsIgnoreCase("-san")) {
+                sans = Arrays.stream(args[1].split("[,]"))
+                        .map(String::trim)
+                        .toArray(String[]::new);
+                mainIndex = 2;
+                expected = 4;
+            } else if (args[2].trim()
+                    .equalsIgnoreCase("-san")) {
+                sans = Arrays.stream(args[3].split("[,]"))
+                        .map(String::trim)
+                        .toArray(String[]::new);
+                expected = 4;
+            }
+        }
+        if (args.length != expected) {
             String canonicalName = CertificateCommand.class.getCanonicalName();
             String exCn = String.format("java %s -cn \"John Smith\"", canonicalName);
-
             String exDn = String.format(
                     "java %s -dn \"cn=John Whorfin, o=Yoyodyne, l=San Narciso, st=California, c=US\"",
                     canonicalName);
             String usage = String.format(
-                    "%nUsage: java %s [-cn <common name>] | [-dn <distinguished name>]%n Examples: %n%s%n%s",
-                    canonicalName, exCn, exDn);
+                    "%nUsage: java %s [-san <subject alt names>] [-cn <common name>] | [-dn <distinguished name>]%n Examples: %n%s%n%s%n%s %s%n%s %s",
```

Comment:
I was not a fan f the way the usage was generated first and didn't want to change it originally. Changed it to align with doc changes done above